bot.go

```diff
@@ -423,6 +423,14 @@ func (bot *BotAPI) GetUpdates(config UpdateConfig) ([]Update, error) {
 	var updates []Update
 	json.Unmarshal(resp.Result, &updates)
 
+	// Every update contains raw JSON which
+	// can include information about other
+	// updates. This is because response is
+	// single for all. Be aware.
+	for _, v := range updates {
+		v.RawUpdateJSON = string(resp.Result)
```

Comment:
`resp.Result` is a byte array (although typed to `json.RawMessage`), instead of converting it to a string you could use it directly.